server/ai_process.go

```diff
@@ -443,7 +538,11 @@ func processAIRequest(ctx context.Context, params aiRequestParams, req interface
 	for {
 		select {
 		case <-cctx.Done():
-			return nil, &ServiceUnavailableError{err: fmt.Errorf("no orchestrators available within %v timeout", processingRetryTimeout)}
+			err := fmt.Errorf("no orchestrators available within %v timeout", processingRetryTimeout)
```

Comment:
@mikezupper, @eliteprox We could also replace this error and the one on line 572 with a more general No capacity message if that makes more sense. My goal is to ensure we can identify when pipelines fail due to insufficient Orchestrators.

Reply:
@mikezupper, @eliteprox, @ad-astra-video we could also split the capacity error in a separate metric but I didn't want to  create unnecessary metrics.